docs/event-dispatcher.md

```diff
@@ -486,3 +508,56 @@ Example:
    ]
 }
 ```
+
+### `POST /attachments/new`
+Delivers information about new AtlasDB attachments processed by the node.
+*   **Triggered by**: Implicitly for all observers.
+*   **Payload Summary**: A JSON array, where each object represents an attachment and includes its index, block details, content hash, contract ID, metadata, transaction ID, and raw content.
+
+*Example Payload:*
+```json
+[
+  {
+    "attachment_index": 123,
+    "index_block_hash": "0xabcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789",
+    "block_height": 75000,
+    "content_hash": "0x112233445566778899aabbccddeeff0011223344",
+    "contract_id": "ST0000000000000000000000000000000000000000.bns",
+    "metadata": "0x0c0000000301000000086e616d6573706163650d0000000362746301000000046e616d650d0000000473617473",
+    "tx_id": "0xfedcba9876543210fedcba9876543210fedcba9876543210fedcba9876543210",
+    "content": "0x48656c6c6f2041746c617321"
+  },
+  {
+    "attachment_index": 124,
+    "index_block_hash": "0xabcdef0123456789abcdef0123456789abcdef0123456789abcdef0123456789",
+    "block_height": 75000,
+    "content_hash": "0xaabbccddeeff00112233445566778899aabbccdd",
+    "contract_id": "ST0000000000000000000000000000000000000000.bns",
+    "metadata": "0x0c0000000301000000086e616d6573706163650d0000000373747801000000046e616d650d00000005737461636b73",
+    "tx_id": "0xfedcba9876543210fedcba9876543210fedcba9876543210fedcba9876543210",
+    "content": "0x416e6f74686572204174746163686d656e74"
+  }
+]
+
+```
+
+### `POST /proposal_response`
+Delivers the validation response for a block proposal submitted to this node (relevant for Nakamoto consensus).
+*   **Triggered by keys**: `"block_proposal"`. **Not** included in `*`.
```

Comment:
if the proposed change in https://github.com/stacks-network/stacks-core/pull/6142/files#r2104782054 is accepted, for consistency this block should be siimlar. 

it not accepted, this comment can be resolved with no changes